s/web/templates/index.scss

```diff
@@ -65,3 +69,13 @@ nub-editor {
 	margin: auto;
 	margin-top: 1em;
 }
+
+nub-gridboard {
+	margin: 1em auto;
+	font-size: 2em;
+}
+
+nub-lookpad {
+	--lookpad-max-width: 15rem;
+	margin: 1rem auto;
+}
```

Comment:
no need for a custom css property when applying something straight onto the custom element `:host` in this way

given the other change of a simple width/height, overriding it from the parent page here would look like:

```css
nub-lookpad {
  width: 20em;
  height: 15em;

  /* margin for demo layout aesthetics is good */
  margin: 1em auto;
}
```